Python/check_paranthesis.py

```diff
@@ -0,0 +1,20 @@
+def is_balanced(s):
+    stack = []
+    for i in range(0, len(s)):
+        c = s[i]
+        if c == "(":
+            stack.append(")")
+        if c == "[":
+            stack.append("]")
+        if c == "{":
+            stack.append("}")
+        elif c == ")" or c == "]" or c == "}":
+            if len(stack) == 0 or stack.pop() != c:
```

Comment:
NOTE: why does stack.pop() doesn't crash. 

HINT: short circuiting in logical operators